papaya-integration-tests/src/test/resources/pl/edu/agh/papaya/tests/projects/Projects.md

```diff
@@ -23,3 +23,25 @@ When authenticated as [ ](- "c:echo=switchUser('peter')"), the current user
 [should not have access](- "c:assert-true=403==tryGetProjectById(#project.id).statusCode")
 to the created project. Their [projects](- "#projects2=getProjects()")
 shall be [empty](- "c:assert-true=#projects2.empty").
+
+## Adding users to the project
+
+Authenticating as [ ](- "c:echo=switchUser('heinz')").
+
+The user [ ](- "c:echo=describeUser('peter')")
+is [added to the project](- "addPeterToProject(#project.id)").
+
+Now, [ ](- "c:echo=switchUser('peter')") should
+[have access to the project](- "c:assert-true=200==tryGetProjectById(#project.id).statusCode").
+
+## Removing users from the project
+
+Authenticating as [ ](- "c:echo=switchUser('heinz')").
+
+The user [ ](- "c:echo=describeUser('peter')")
+is [removed from the project](- "removePeterFromProject(#project.id)").
+
+When authenticated as [ ](- "c:echo=switchUser('peter')"), the current user
+[should not have access](- "c:assert-true=403==tryGetProjectById(#project.id).statusCode")
+to the created project. Their [projects](- "#projects2=getProjects()")
+shall be [empty](- "c:assert-true=#projects2.empty").
```

Comment:
To jeszcze można coś o tym, że jak email nie istnieje, to nie da się tego kogoś dodać, oraz że nie-admin nie może nikogo dodać/dezaktywować, a admin może

Reply:
Czy moja sugestia testów to uratowała, czy sam na to wpadłeś?

Reply:
Już wiedziałem o tym problemie wcześniej 😖